src/main/kotlin/no/nav/syfo/api/v2/domain/GodkjennPlanVarsel.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.api.v2.domain
+
+data class GodkjennPlanVarsel(
+    val erSykmeldt: Boolean,
```

Comment:
Kan vi utlede om det er arbeidstaker eller arbeidsgiver utfra hvem som er innlogget, kanskje?

Reply:
Og id kan kanskje være en parameter?

Reply:
@kaja-nav Det er et POST kall som gjøres fra oppfolgingsplan-frontend. Vanlig er å ha parametere i body
som JSON. Vi utleder sykmeldt i frontend for å slippe sjekken her, basert på om de er sykmeldt eller arbeidstaker
"pagen".

Reply:
@kaja-nav Hva tenker du på med "parameter"? At den (oppfolgingsplanId) skal ligge i som path-param i URL'en? I så fall, hvorfor bør den ikke være en del av body'en?